public/app/features/scenes/services/UrlSyncManager.ts

```diff
@@ -63,7 +63,7 @@ export class UrlSyncManager {
       }
 
       if (Object.keys(mappedUpdated).length > 0) {
-        locationService.partial(mappedUpdated, false);
+        locationService.partial(mappedUpdated, true);
```

Comment:
Agree with @ivanortegaalba. I don't think we need to fix this now, but in the future, I think this will be necessary. 

Thinning especially about drill-down scenarios. I know there are two ways of approaching it, either presenting a completely new drill-down scene or conditionally rendering parts of the current scene if i.e. certain variables are applied. In the second scenario, history push will be essential not only for navigating but also for sharing narrowed-down scenes.